src/utils/sampleBackgroundPoints.ts

```diff
@@ -0,0 +1,53 @@
+import { Image } from '../Image';
+import { Mask } from '../Mask';
+import { Point } from '../geometry';
+
+interface SampleBackgroundPointsOptions {
+  numberOfRows?: number;
+  numberOfColumns?: number;
+  kind?: 'black' | 'white';
+}
+/**
+ * Returns a sample of points that belongs to background.
+ * @param image - Image to sample points from.
+ * @param mask - Mask to check if the point belongs to background.
+ * @param options - SampleBackgroundPointsOptions.
+ * @returns Array of points.
+ */
+export function sampleBackgroundPoints(
+  image: Image,
+  mask: Mask,
```

Comment:
If the whole `options` parameter is undefined or just the number of rows and columns for the grid? Because we need the mask to sample the background anyway. And since we do, isn't it better to keep it as a standalone parameter(the way it is now)?

Reply:
Number of rows and columns should probably equal to image's height and width respectively, indeed.

Reply:
@lpatiny?

Reply:
If we don't have a mask with consider the grid, 10 x 10 points, as the background.